public/styles/Footer.scss

```diff
@@ -0,0 +1,15 @@
+@import '~bootstrap/scss/functions';
+@import '~bootstrap/scss/variables';
+@import '~bootstrap/scss/mixins';
+
+footer {
+  font-size: 0.8em;
+  bottom: 0;
+  position: absolute;
+  width: 100%;
+  height: 60px;
```

Comment:
I think the team agree to go with `em` instead of `px`. You can convert px to em using 16px:1 em.

Reply:
I have fixed this.